### ISSUE TITLE
Reuploaded 3 times with new code

### ISSUE THREAD

**Issue by tenumars** [maximedrn/opensea-automatic-bulk-upload-and-sale#109]
Somehow it uploaded the same nft with the same metadata 3 times overnight when I left it running. It uploaded 963, but then restarted and reuploaded the other ones. What caused that and how can I fix it? Also how can I now delete all of those copies in a mass amount? The recaptcha thing works though, I can confirm

**Comment by maximedrn** [comment]
With version 1.5.0 or 1.5.1?

**Comment by maximedrn** [comment]
Version 1.5.1 should correct this problem.

**Comment by tenumars** [comment]
even with 1.5.1

**Comment by maximedrn** [comment]
Send me a video because it seems that I fix this issue.

**Comment by tenumars** [comment]
sorry, its through my phone with remote desktop, numbers 1-202 have already been uploaded

**Comment by maximedrn** [comment]
You should delete the line of already downloaded NFTs. If you stop the robot during the download, it does not remember anything about the download and the file. I need to add logs in the future.
Doesn't the robot loop on the same NFT?

**Comment by maximedrn** [comment]
> it is now duplicating them <img alt="image" width="992" src="https://user-images.githubusercontent.com/98669318/154539171-d89157c5-b1ca-4598-880d-b557fb3d36d4.png">

You need to delete them one by one :/ I need to create an NFT editor and deleter.

**Comment by tenumars** [comment]
ok, thanks, is there a way to mass transfer them to coinbase wallet? like reserving for a specific buyer @0 eth? or something else?

**Comment by maximedrn** [comment]
Yes, you can do this, you may need to modify the code to remove the condition for a price less than 0 (I need to modify this as Ethereum allows a zero price, however I don't know if Polygon allows this).

**Comment by tenumars** [comment]
from the metadata?

**Comment by maximedrn** [comment]
Yes, the bot reads your metadata file line by line.
